### PR TITLE
feat: Added task to collect repository traffic

### LINKED ISSUE #1468: Extend Augur to collect Github Traffic Metrics
**Is your feature request related to a problem? If so, please describe the problem:**
For project https://github.com/aicoe-aiops/ocp-ci-analysis, we wish to use Augur to collect and analyze Github traffic related information.
Does Augur currently scrape Traffic information such as Referring sites and Views per page from Git repositories? 

**Potential solutions:**
Extend Augur to collect [Github traffic information](https://docs.github.com/en/repositories/viewing-activity-and-data-for-your-repository/viewing-traffic-to-a-repository) using Github API.

**Additional context:**
related: https://github.com/aicoe-aiops/ocp-ci-analysis/issues/382

### PR DESCRIPTION
**Pull request by meetagrawal09**
**Description**
- The PR adds a task to collect Github repository traffic data (specifically repository clone history data)
- Github API docs - https://docs.github.com/en/rest/metrics/traffic#get-repository-clones
This PR fixes #1468 , #1373 

**Signed commits**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Review by sgoggins — CHANGES_REQUESTED** [review]
This is a pretty fallow week for us, but I think there are a couple schema migration related changes requested you could work on, or ask for help with. 

@ABrain7710 : Can you look at this PR?

File: augur/application/db/models/augur_data.py [hunk 1]
```diff
@@ -3163,3 +3163,31 @@ class PullRequestReviewMessageRef(Base):
     msg = relationship("Message")
     pr_review = relationship("PullRequestReview")
     repo = relationship("Repo")
+
+
```
[the remark this answers is not in the document]
Comment (meetagrawal09) [reply augur/application/db/models/augur_data.py:3167]: I have auto-generated the schema migration script from the model `RepoClass` that I had defined, using the command `alembic revision --autogenerate --rev-id "3" -m "traffic additions"` 

Do let me know if I need to roll-back changes to the `augur_full.sql` and if the model class in `augur_data.py` needs to be placed in a separate file

File: augur/application/schema/augur_full.sql [hunk 2]
```diff
@@ -2777,6 +2777,35 @@ CREATE TABLE augur_data.working_commits (
 
 ALTER TABLE augur_data.working_commits OWNER TO augur;
 
+--
+-- Name: repo_clones_id_seq; Type: SEQUENCE; Schema: augur_data; Owner: augur
```
[the remark this answers is not in the document]
Comment (meetagrawal09) [reply augur/application/schema/augur_full.sql:2781]: @sgoggins can you please guide me as to what tool is used for schema migration scripts here or if the alembic version file generated would be the one sufficient.

**Comment by sgoggins** [comment]
@ABrain7710 : I need to sit down with you to finish merging this. There's a schema change file that's out of order with work that happened since this was created.

**Pull request merged**